python/GenerateE2Authcode.py

```diff
@@ -0,0 +1,63 @@
+import hashlib
+import random
+
+"""
+This is an example how to calculate authcode for Paytrail E2 payment interface.
+More information about authcode can be found in: https://paytraildocs.netlify.app/payments/e2-interface/authcode/
```

Comment:
Let's add the production domain here in advance. Or this could be removed for now and added again later.

```suggestion
More information about authcode can be found in: https://docs.paytrail.com/payments/e2-interface/authcode/
```

Reply:
Changed to current live documentation url